### PR TITLE
HPC : Guide update life cycle policy

### PR DESCRIPTION
**Pull request by pg-ovh**
Update lifecycle policy for HPC VMware on OVHcloud offers.

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md
Comment (Y0Coss) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md]: There is no space in English before punctuation signs. Please revert this change on lines 18, 21, 22, 129 for all guides except fr-fr and fr-ca.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md
Comment (Y0Coss) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md]: wrong link on all versions except fr-fr and fr-ca
It should be `(/links/community)`

**Review by Y0Coss — CHANGES_REQUESTED** [review]

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
Waiting for the review from @franckcosta to merge the PR.
**Note**: I'd recommend switching the dates format to the international format YYYY-MM-DD

**Comment by pg-ovh** [comment]
Thanks @Y0Coss it's all good with @franckcosta.

**Comment by Y0Coss** [comment]
Thank you @pg-ovh for your contribution

**Pull request merged**
